src/classes/asettings.h

```diff
@@ -88,6 +89,10 @@ class ASettings {
     static QVariant read(const NewFlight key);
     static void write(const NewFlight key, const QVariant &val);
 
+#if QT_VERSION > QT_VERSION_CHECK(5, 9, 0)
```

Comment:
What is the purpose of this function? It is not used anywhere in the application.

If we need it, we can implement it, but for now I would rather not have a function around which is:
- unused
- potentially troublesome

A note on the logic: Just because it is not good for 5.9 doesn't mean it's good for 5.10 or 5.11, we only know it's okay for 5.12 and higher.

Reply:
I dont know why we have it so we can just remove it for now. I dont remember its use.